package.json

```diff
@@ -36,6 +36,7 @@
     "jsx-loader": "^0.13.2",
     "moment": "^2.22.1",
     "mustache": "^2.3.0",
+    "neo-async": "^2.5.1",
```

Comment:
@KipSigei Wait what's this?

Reply:
not really sure, I got a `missing 'neo-async' module` error when compiling so I added it